src/Compatibility/Core/src/Windows/ButtonRenderer.cs

```diff
@@ -274,27 +274,42 @@ static StackPanel CreateContentContainer(Button.ButtonContentLayout layout, WIma
 				case Button.ButtonContentLayout.ImagePosition.Top:
 					container.Orientation = Orientation.Vertical;
 					image.Margin = WinUIHelpers.CreateThickness(0, 0, 0, spacing);
+#pragma warning disable RS0030 // Do not use banned APIs; Panel.Children is banned for performance reasons.
 					container.Children.Add(image);
 					container.Children.Add(textBlock);
+#pragma warning restore RS0030 // Do not use banned APIs
```

Comment:
@mattleibow should we instead just not use `BannedApiAnalyzer` in `src/Compatibility`? Maybe we don't care as much about this project going forward?